Misc/NEWS

```diff
@@ -806,6 +806,8 @@ Documentation
 Build
 -----
 
+- Issue #29572: Update Windows build and OS X installers to use OpenSSL 1.0.2k.
```

Comment:
Yes, that's up to @ned-deily.  I went ahead and copied the message from our 1.0.2j update assuming that it will be done anyway :)